systori/apps/project/api.py

```diff
@@ -54,6 +65,28 @@ def get_serializer_class(self):
                 return self.action_serializers[self.action]
         return super(ProjectModelViewSet, self).get_serializer_class()
 
+    project_not_found_response = openapi.Response(
+        "Project not found",
+        openapi.Schema(
+            "Project not found",
```

Comment:
did you make sure that this still gets translated? IIRC that was the reason to use the server response instead of generating the response client side...

Reply:
Or is this just to describe thins in swagger.yaml?

Reply:
Just to describe the schema